src/RSocketSetup.h

```diff
@@ -22,7 +24,7 @@ class RSocketStats;
 class RSocketSetup {
  public:
   RSocketSetup(
-      std::shared_ptr<FrameTransport> frameTransport,
+      yarpl::Reference<FrameTransport> frameTransport,
```

Comment:
this is implied by the FrameTransport implementing yarpl::Subscriber which is a Reference type.
Please keep in mind the FrameTransport type in on its way out.